SimpleFSDM-SET_COMMAND/commands/model/primary_key_value.py

```diff
@@ -0,0 +1,13 @@
+from index import *
+
+
+class PrimaryKeyIndex(Index):
+    def __init__(self, primary_key_name, table_metadata):
+        super().name = primary_key_name
+        super().__path = table_metadata.get_path()
+

```

Comment:
you can use `super().__init__(primary_key_name, table_metadata)` instead of those two lines